packages/jest-phabricator-coverage/src/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
```

Comment:
this should be using `@flow`. You can find a ton of existing types in the `types/` folder but you may need to create a few more.